src/Compilers/CSharp/Test/Symbol/SymbolDisplay/SymbolDisplayTests.cs

```diff
@@ -5510,7 +5510,6 @@ public class C
             var compB = CreateVisualBasicCompilation(GetUniqueName(), "", referencedAssemblies: new[] { MscorlibRef, refA });
             compB.VerifyDiagnostics();
             // From VB symbols.
-            //PROTOTYPE(refReadonly): metadata emit and VB NYI
             //RefReadonlyReturnInternal(compB);
```

Comment:
Are we going to leave this commented code?

Reply:
The bug refers to the commented code, so it is easier to find. I think, considering that this is a test, having a comment is ok.

---
In reply to: [139489534](https://github.com/dotnet/roslyn/pull/21982#discussion_r139489534) [](ancestors = 139489534)